tests/containers/integration-test-snapshot

```diff
@@ -17,6 +17,7 @@ RUN dnf install \
         bluechi-ctl-debuginfo \
         bluechi-selinux \
         python3-bluechi \
+        procps-ng \
```

Comment:
This was already added in the `integration-test-base` image and can be removed here.

Reply:
done, removed